openbb_terminal/stocks/stocks_controller.py

```diff
@@ -454,7 +455,7 @@ def call_candle(self, other_args: List[str]):
                 if ns_parser.raw:
                     qa_view.display_raw(
                         data=self.stock,
-                        sortby=ns_parser.sort,
+                        sortby=" ".join(ns_parser.sort),
```

Comment:
Im confused.  The above makes the sort columns have no spaces or an underscore.  What is the case for when multiple words would be supplied?